strutil/pathiter_test.go

```diff
@@ -35,18 +35,35 @@ func (s *pathIterSuite) TestPathIteratorEmpty(c *C) {
 	c.Assert(iter, IsNil)
 }
 
+func (s *pathIterSuite) TestCurDir(c *C) {
+	iter, err := strutil.NewPathIterator(".")
```

Comment:
shouldn't we also have tests about ./foo/bar ?

Reply:
I added the test. "./foo/bar" is an unclean path so trying to create an iterator results in an error. The current directory is a special case that is clean by itself, but adding the dot in general is unclean.